File: решения/java/src/lab17/Order.java
package lab17;

/**
 * Created by Александр on 20.07.2017.
 */
public class Order {
    private String id;
    private String date;
    private String productId;
    private int quantity;
    private String clientId;
    private String employeeId;
    private boolean finished;

    public String getId() {
        return id;
    }

    public String getDate() {
        return date;
    }

    public String getProductId() {
        return productId;
    }

    public int getQuantity() {
        return quantity;
    }

    public String getClientId() {
        return clientId;
    }

    public String getEmployeeId() {
        return employeeId;
    }

    public boolean isFinished() {
        return finished;
    }

    public Order(String id, String date, String productId, int quantity, String clientId, String employeeId, boolean finished) {

        this.id = id;
        this.date = date;
        this.productId = productId;
        this.quantity = quantity;
        this.clientId = clientId;
        this.employeeId = employeeId;
        this.finished = finished;
    }
}

File: решения/cs/Classes/lab17/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Classes.lab17
{
    public class Order
    {
        public String Id { private set; get; }
        public String Date { private set; get; }
        public String ProductId { private set; get; }
        public int Quantity { private set; get; }
        public String ClientId { private set; get; }
        public String EmployeeId { private set; get; }
        public bool IsFinished { private set; get; }

        public Order(String id, String date, String productId, int quantity,
            String clientId, String employeeId, bool isFinished)
        {
            Id = id;
            Date = date;
            ProductId = productId;
            Quantity = quantity;
            ClientId = clientId;
            EmployeeId = employeeId;
            IsFinished = isFinished;
        }
    }
}
